### PR TITLE
EM-4110: remove old un-used ase tf code

### PR DESCRIPTION
**Pull request by rishikrsharma**
### JIRA link (if applicable) ###
EM-4110

### Change description ###
remove old un-used ase tf code

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: infrastructure/main.tf [hunk 1]
```diff
@@ -27,107 +26,27 @@ locals {
   nonPreviewVaultName = "${var.shared_product}-${var.env}"
   vaultName           = (var.env == "preview" || var.env == "spreview") ? local.previewVaultName : local.nonPreviewVaultName
 
-  sharedAppServicePlan       = "${var.shared_product}-${var.env}"
-  sharedASPResourceGroup     = "${var.shared_product}-shared-${var.env}"
+  sharedAppServicePlan   = "${var.shared_product}-${var.env}"
```
Comment (jasonpaige) [inline infrastructure/main.tf:29]: this can go now

File: infrastructure/main.tf [hunk 2]
```diff
@@ -27,107 +26,27 @@ locals {
...
+  sharedASPResourceGroup = "${var.shared_product}-shared-${var.env}"
```
Comment (jasonpaige) [inline infrastructure/main.tf:30]: this can go too

**Review by jasonpaige — APPROVED** [review]

File: infrastructure/main.tf [hunk 3]
```diff
@@ -27,107 +26,33 @@ locals {
...
-    # Document Storage
-    STORAGEACCOUNT_PRIMARY_CONNECTION_STRING   = data.azurerm_key_vault_secret.dm_store_storageaccount_primary_connection_string.value
-    STORAGEACCOUNT_SECONDARY_CONNECTION_STRING = data.azurerm_key_vault_secret.dm_store_storageaccount_secondary_connection_string.value
-    STORAGE_CONTAINER_DOCUMENT_CONTAINER_NAME  = azurerm_storage_container.document_container.name
-
-    TEMP_FORCE_REDEPLOY = "1"
-
-    TASK_ENV = "documentTaskLock-${var.env}"
-  }
+resource "azurerm_resource_group" "rg" {
```
Comment (jasonpaige) [inline infrastructure/main.tf:30]: you don't need this resource group either - nothing get's deployed there

File: infrastructure/main.tf (hunk 1 above)
Comment (rishikrsharma) [reply infrastructure/main.tf]: removed.

File: infrastructure/main.tf (hunk 2 above)
Comment (rishikrsharma) [reply infrastructure/main.tf]: removed

File: infrastructure/main.tf (hunk 3 above)
Comment (rishikrsharma) [reply infrastructure/main.tf]: removed.

**Review by timja — APPROVED** [review]
lgtm, inside the RG there's just an app insights that doesn't appear to be used and an unused traffic manager profile

**Pull request merged**